modules/users/server/controllers/admin.server.controller.js

```diff
@@ -88,6 +88,49 @@ exports.list = function (req, res) {
   });
 };
 
+exports.unapprovedList = function(req, res) {
+  User
+    .find({ approvedStatus: false })
+    .sort('-created')
+    .populate('user', 'displayName')
+    .exec(function(err, unapprovedUsers) {
+      if (err) {
+        return res.status(422).send({
+          message: errorHandler.getErrorMessage(err)
+        });
+      }
+
+      res.json(unapprovedUsers);
+    });
+}
+
+exports.changeToAccepted = function (req, res) {
```

Comment:
try to have this function be named similar to the one in the client side controller because it is easier to read that way